src/components/Input/SearchField.stories.tsx

```diff
@@ -38,6 +38,7 @@ export default {
     readOnly: { control: "boolean" },
     orientation: { control: "inline-radio", options: ["horizontal", "vertical"] },
     dir: { control: "inline-radio", options: ["start", "end"] },
+    isFilter: { control: "boolean" },
```

Comment:
I think instead of this we could allow the icon as a prop

Reply:
Why? It seems a bit open to me, I'm not sure we want people using random icons like star or heart for this? It's a search field.

Reply:
I was thinking of the same, In case we need a search field with different icon when we migrate other applications in our org then this might be needed